Code/EditorPlugins/Scene/EditorPluginScene/Actions/SceneActions.cpp

```diff
@@ -332,6 +332,33 @@ void ezSceneAction::Execute(const ezVariant& value)
         }
       }
 
+
+      // Convert collections
+      {
+        EZ_PROFILE_SCOPE("ConvertCollections");
+        ezAssetCurator* pCurator = ezAssetCurator::GetSingleton();
+        ezSet<ezUuid> collections;
+        {
+          ezAssetCurator::ezLockedAssetTable allAssets = pCurator->GetKnownAssets();
+
+          ezTempHashedString sCollection = "Collection";
+          for (auto it : *allAssets)
+          {
+            if (it.Value()->m_Info->m_sAssetsDocumentTypeName == sCollection)
```

Comment:
Instead of hardcoding this for collections, we could also add a virtual function "NeedsTransformOnSceneExport". Then in the future other assets could also utilize this.
Not important though, can also be changed when the need really arises.

Reply:
Added an #TODO_ASSET to this code for future reference.